build.gradle.kts

```diff
@@ -30,6 +19,8 @@ allprojects {
 }
 
 subprojects {
+    // TODO libs doesn't resolve if we do this
+    // apply(plugin = libs.plugins.ktlint.get().pluginId)
     apply(plugin = "org.jlleitschuh.gradle.ktlint")
```

Comment:
version catalogs feeling not totally ready yet for plugins. The IDE seemed to like it but it fails to resolve when you build.